decode_morse_message.rb

```diff
@@ -0,0 +1,27 @@
+def decode_char(char)
+  decoder = { '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+              '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+              '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+              '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '.----' => '1',
+              '..---' => '2', '...--' => '3', '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7',
+              '---..' => '8', '----.' => '9', '-----' => '0', ' ' => '', '/' => '/' }
+  decoder[char]
+end
+
+def separator(word)
+  word.split('   ')
+end
+
+def decode_msg(message)
+  result = ''
+  message.each do |n|
+    word = n.split
+    word.each do |m|
+      result += decode_char(m)
+    end
+    result += ' '
+  end
+  puts(result)
+end
```

Comment:
- [OPTIONAL] Nice implementation here, it works as expected, but one of the objectives of this project is to create reusable methods(that is to make our code DRY), it will be nice to tweak your method to reuse the methods created  above so that;
* ` it splits input sentences by 3 spaces (word delimiter in Morse code)`
* `it iterates through all words from the input sentence`
* `it reuses the `#decode_word` method to translate a single encoded-word into a word (DRY rule)`
* `it combines translated words into a whole sentence to be returned as the final output`